src/System.Management.Automation/FormatAndOutput/common/Utilities/Mshexpression.cs

```diff
@@ -159,27 +159,41 @@ public List<PSPropertyExpression> ResolveNames(PSObject target, bool expand)
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-linux`
Select-Object DRT basic functionality.Select-Object with empty script block property should throw
```powershell
Expected an exception, with FullyQualifiedErrorId 'EmptyScriptBlockAndNoName,Microsoft.PowerShell.Commands.SelectObjectCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Select-Object.Tests.ps1: line 131
131:             Should -Throw -ErrorId "EmptyScriptBlockAndNoName,Microsoft.PowerShell.Commands.SelectObjectCommand" -PassThru
```